src/menu/src/MenuItem.js

```diff
@@ -13,7 +13,11 @@ class MenuItem extends React.PureComponent {
      * Element type to use for the menu item.
      * For example: `<MenuItem is={ReactRouterLink}>...</MenuItem>`
      */
-    is: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
+    is: PropTypes.oneOfType([
```

Comment:
Possibly do `Pane.propTypes.is`?

Reply:
Yeah, that'd be a little bit more accurate.

Reply:
Unfortunately I don't think it's possible to access `Pane.propTypes.is` (says Pane.propTypes is undefined). It might be possible to do once Pane is a hook however.

Reply:
You could use `Box.propTypes` like Pane does.